udms/e2e/firestore/import_db_cloud.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @fileoverview Populates firestore db with JSON object in a given file
+ */
+
+// Point json_data_source to the json file with data to be populated into the DB
+// Point db_collection to the collection name/topmost field of json object being
+// populated.
+// If script fails with permission issues, run:
+// 'gcloud auth application-default login'
+
+const firebase = require("firebase-admin");
+require("firebase/firestore");
```

Comment:
Is this required necessary? Seems like it’s used anywhere.

Reply:
Removed.